neuralop/data/datasets/pt_dataset.py

```diff
@@ -96,7 +101,9 @@ def __init__(self,
 
         # optionally subsample along data indices
         ## Input subsampling 
-        input_data_dims = data["x"].ndim - 1
+        input_data_dims = data["x"].ndim - 2 # batch and channels
+        if channels_squeezed:
+            input_data_dims += 1
```

Comment:
That's a little convoluted - could we potentially just do the unsqueeze first?

Reply:
Yep, changed